testData/solutions/win32.json

```diff
@@ -1318,5 +1318,91 @@
+        "contexts": {
+            "OS": [{
+                    "id": "win32",
+                    "version": ">=5.0"
+            }]
+        },
+        "settingsHandlers": [
+           {
+                "type": "gpii.settingsHandlers.webSockets",
+                "options": {
+                    "path": "org.mozilla.cloud4firefox"
+                },
+                "capabilities": [
+                    "applications.org\\.mozilla\\.cloud4firefox.id"
+                ],
+                "capabilitiesTransformations": {
+                    "screenReaderTTSEnabled":"http://registry\\.gpii\\.net/common/screenReaderTTSEnabled",
+                    "magnifierEnabled": "http://registry\\.gpii\\.net/common/magnifierEnabled",
+                    "magnification": "http://registry\\.gpii\\.net/common/magnification",
+                    "fontSize": {
+                        "transform":{
+                            "type":"gpii.transformer.quantize",
+                            "inputPath":"http://registry\\.gpii\\.net/common/fontSize",
+                            "ranges":[
+                                {
+                                    "upperBound":14,
+                                    "output":"M"
+                                },
+                                {
+                                    "upperBound":24,
+                                    "output":"L"
+                                },
+                                {
+                                    "output":"XL"
+                                }
+                            ]
+                        }
+                    },
+                    "backgroundColour": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/highContrastTheme",
+                            "options": {
+                                "white-black": "#000000",
+                                "black-white": "#FFFFFF",
+                                "yellow-black": "#000000",
+                                "black-yellow": "#FFFF00"
+                            }
+                        }
+                    },
+                    "foregroundColour": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/highContrastTheme",
+                            "options": {
+                                "white-black": "#FFFFFF",
+                                "black-white": "#000000",
+                                "yellow-black": "#FFFF00",
+                                "black-yellow": "#000000"
+                            }
+                        }
+                    },
+					
+					"highContrastEnabled": "http://registry\\.gpii\\.net/common/highContrastEnabled",
```

Comment:
No, it is not possible. You have committed tabs rather than spaces in these files - you need to set your editor so that it produces an indent of 4 spaces and does not try to reconvert back to tabs. See this line in your committed file:

https://github.com/GutiX/universal/blob/Firefox-Extension/testData/solutions/win32.json#L1389